PythonCodeConventions.md

```diff
@@ -49,8 +53,49 @@ y = fib(2.5)
 In this case, mypy will inform that the call using the floating point value is not valid:
 __mypytest.py:10: error: Argument 1 to "fib" has incompatible type "float"; expected "int"__
 
-Checker script(s)
-=======
+The following construct is _only_ allowed in case of unavoidable circular dependencies, otherwise regular import statements MUST be used.
+``` python
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from [package] import [class]
+```
+## Line breaking
+Indentation of function definitions and function calls that exceed the soft line length (PEP8: 80, Ultimaker: 120 characters) is described in PEP8 https://www.python.org/dev/peps/pep-0008/#indentation. From the suggested set, the following SHOULD be applied:
+
+When a function definition exceeds the maximum line length, wrap the lines as such:
+``` python
+def long_function_name(
```

Comment:
I don't mind keeping it like this, but would slightly prefer putting the first bracket `(` on a newline, because that is congruent to how `{` is treated when starting blocks (function, if, etc..).

Reply:
Just realized that this wouldn't really work for Python because you'd have to add line breaks every time. So nevermind :-)